### PR TITLE
Implement #225 and #128

### PR DESCRIPTION
**Pull request by kgarsuta**

### CODE REVIEW

**Comment by projectdelphai** [comment]
I'll take a look at this commit when I go home, should work fine I'd think though. I'll merge it if there's no problems!

**Comment by paddatrapper** [comment]
Looks good to me :+1:

File: json/original_data/items.json [hunk 1]
```diff
@@ -261,6 +261,18 @@
...
+    },
+	"katana": {
+      "type": "weapon",
+      "name": "Katana",
+      "description": "A long sword once used by a legendary Japanese samurai",
+      "position": "rightHand",
+      "level": 3,
+      "properties": {
+        "damage": "+50",
+		"weight": "3",
```
Comment (paddatrapper) [inline json/original_data/items.json]: Indenting here is a little off

File: json/original_data/items.json [hunk 2]
```diff
@@ -261,6 +261,18 @@
...
+        "value": "0"
```
Comment (paddatrapper) [inline json/original_data/items.json]: Why do you not give the Katana any monetary value?

Comment (kgarsuta) [inline json/original_data/items.json]: I belive @BusyBob was testing out a specific feature

**Comment by projectdelphai** [comment]
Alright, sorry about the long delay, had finals to finish up. Going to try and work a little everyday now that I have more free time. Time to learn how to code on Windows too since I don't have my linux comp on me :D Good job on the commit although the project.txt file was not necessary. I removed it from the commit. Merged and closed.

**Comment by kgarsuta** [comment]
@projectdelphai Good luck =) Visual studio is a lot of fun.

**Comment by projectdelphai** [comment]
actually i downloaded and installed git, apache-ant, and java and then have been just coding everything in git bash. Probably not the best way to do it but it was pretty seamless lol. It's been working so far haha

**Pull request merged**